deployments/kubernetes/charts/clowder2/values.yaml

```diff
@@ -168,25 +168,31 @@ rabbitmq:
     #storageClass: csi-cinder-sc-retain
 
 # ----------------------------------------------------------------------
-# EXTRACTORS
+# DEFAULT EXTRACTOR SERVICES
 # ----------------------------------------------------------------------
+message:
+  image:
+    repository: clowder/clowder2-messages
+    tag: main
+    pullPolicy: Always
+
+heartbeat:
+  image:
+    repository: clowder/clowder2-heartbeat
+    tag: main
+    pullPolicy: Always
 
+
+# ----------------------------------------------------------------------
+# EXTRACTORS
+# ----------------------------------------------------------------------
 ## list of extractors to be installed. Each extractor should have the
 ## following fields:
 ## enabled      : true/false - should the extractor be installed
 ## image        : dokeer/image:tag - the extractor docker image
 ## replicaCount : number - the number of this extractor to install
 ## env          : additional environment variables to pass to extractor.
 extractors:
-#  file-digest:
```

Comment:
any specific reason to remove digest?

Reply:
we haven't converted to be v2 compatible yet.. maybe I should create a task for that.